tests/storages_tests/test_with_server.py

```diff
@@ -2,11 +2,14 @@
 import os
 from typing import Sequence
 from typing import Tuple
+from typing import Union
 
 import numpy as np
 import pytest
 
 import optuna
+from optuna.storages import RDBStorage
+from optuna.storages._redis import RedisStorage
```

Comment:
A nit, but `._redis` can be removed.
```suggestion
from optuna.storages import RedisStorage
```

Reply:
Fixed also `tests/storages_tests/test_storages.py`.

Reply:
Thanks!